pkg/test/framework/components/istio/operator.go

```diff
@@ -152,8 +154,13 @@ func deploy(ctx resource.Context, env *kube.Environment, cfg Config) (Instance,
 		return nil, err
 	}
 
-	// For multicluster, create and push the CA certs to all clusters to establish a shared root of trust.
 	if env.IsMulticluster() {
+		// Only setup namespaces ourselves in multicluster, other tests rely on the default namespace config
```

Comment:
To get around this we can change the way tests that depend on `ClaimSystemNamespace[OrFail]` work by having them get the proper system namespace by passing a `SetupConfigFn` to `istio.Setup` and getting a reference to the config from there to avoid using `DefaultConfig` and possibly eliminating `ClaimSystemNamespace` altogether.